### PR TITLE
Lps 202947 master

### PR DESCRIPTION
**Pull request by SamZiemer**
https://liferay.atlassian.net/browse/LPS-202947

Renaming configuration property to prevent duplicate properties from causing issues.

/cc @robertoDiaz, @adolfopa

### CODE REVIEW

**Comment by SamZiemer** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

File: modules/apps/adaptive-media/adaptive-media-image-impl-test/src/testIntegration/java/com/liferay/adaptive/media/image/internal/scaler/test/AMImageScalerRegistryImplTest.java [hunk 1]
```diff
@@ -312,7 +312,7 @@ private ServiceRegistration<AMImageScaler> _registerAMImageScaler(
 		Dictionary<String, Object> properties = new Hashtable<>();
 
 		properties.put("service.ranking", serviceRanking);
-		properties.put("supportedMimeTypes", mimeType);
+		properties.put("mimeTypes", mimeType);
```
Comment (adolfopa) [inline modules/apps/adaptive-media/adaptive-media-image-impl-test/src/testIntegration/java/com/liferay/adaptive/media/image/internal/scaler/test/AMImageScalerRegistryImplTest.java:315]: You need to sort this alphabetically.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/adaptive-media/adaptive-media-image-impl-test/src/testIntegration/java/com/liferay/adaptive/media/image/internal/scaler/test/AMImageScalerRegistryImplTest.java (hunk 1 above)
Comment (SamZiemer) [reply modules/apps/adaptive-media/adaptive-media-image-impl-test/src/testIntegration/java/com/liferay/adaptive/media/image/internal/scaler/test/AMImageScalerRegistryImplTest.java]: Thanks @adolfopa, I've added a new commit fixing that.

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by SamZiemer** [comment]
These failure seems unrelated, is this eligible to be force forwarded?

**Comment by Jeremy-Ch** [comment]
ci:forward:force

**Pull request closed without merging**